File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    point = 0
    if a != b and b != c and c != a:
        print(1)
        point = a+b+c
    elif a == b and b == c and b == c:
        print(3)
        point = (a+b+c)*(a**2+b**2+c**2)*(a**3+b**3+c**3)
    else:
        print(2)
        point = (a+b+c)*(a**2+b**2+c**2)
          
    return point

File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        
        if(a!=b && b!=c && c!=a) {
            answer = a+b+c;
        }else if(a==b && b==c && c==a) {
            answer = (a+b+c)*(getPow(a, 2)+getPow(b, 2)+getPow(c, 2))*(getPow(a, 3)+getPow(b, 3)+getPow(c, 3));
        }else {
            answer = (a+b+c)*(getPow(a, 2)+getPow(b, 2)+getPow(c, 2));
        }
        
        return answer;
    }
    
    public int getPow(int number, int powNumber) {
        return (int)Math.pow(number, powNumber);
    }
}